distutils/compat/__init__.py

```diff
@@ -0,0 +1,15 @@
+from __future__ import annotations
+
+from .py38 import removeprefix
+
+
+def consolidate_linker_args(args: list[str]) -> str:
+    """
+    Ensure the return value is a string for backward compatibility.
+
+    Retain until at least 2024-04-31. See pypa/distutils#246
```

Comment:
@jaraco Just to double checking... do you really meant 2024-04-31 here? That's about just two weeks from today, not six months!

Reply:
I meant 2025. Fixed in e5268b516.